src/pages/[platform]/build-a-backend/data/data-modeling/relationships/index.mdx

```diff
@@ -114,24 +114,16 @@ Amplify.API.mutate(ModelMutation.create(team),
 <InlineFilter filters={["swift"]}>
 
 ```swift
-Task {
-    do {
-        let team = Team(mantra: "Go Frontend!")
-        let createTeamResult = try await Amplify.API.mutate(request: .create(team))
-        guard case .success = createTeamResult else {
-            print("API response: \(createTeamResult)")
-            return
-        }
-
-        let member = Member(name: "Tim", team: try createTeamResult.get()) // Directly pass in the post instance
-        let createMemberResult = try await Amplify.API.mutate(request: .create(member))
-        guard case .success = createMemberResult else {
-            print("API response: \(createMemberResult)")
-            return
-        }
-    } catch {
-        print("Create team or member failed", error)
-    }
+do {
+    let team = Team(mantra: "Go Frontend!")
+    let createdTeam = try await Amplify.API.mutate(request: .create(team)).get()
+
+    let member = Member(
+        name: "Tim",
+        team: createdTeam) // Directly pass in the team instance
+    let createdMember = try await Amplify.API.mutate(request: .create(member))
+} catch {
+    print("Create team or member failed", error)
 }
```

Comment:
<img width="1006" alt="image" src="https://github.com/aws-amplify/docs/assets/1365977/8c8ccea6-f89b-4fdd-83d7-4cf96ee88a4d">


These changes include 
- removing the guard statements and letting it get caught in the catch block by using `get()`.
- removing the Task, seems unnecessary extra code to display